tests/integration/test_case.go

```diff
@@ -338,6 +339,41 @@ type Request struct {
 	ExpectedError string
 }
 
+// GenerateDocsFromSchema is an action that will trigger generation of documents for the given schema.
+type GenerateDocsFromSchema struct {
```

Comment:
suggestion: I do think these actions do too much, and could be simplified a fair bit.  It does not make sense to me to force consumers of these to use them over the existing `SchemaUpdate` action, when they should not be mutually exclusive.  The current setup also makes it painful to use them on a patched schema.

I would want the tests to look something like this:
```
testUtils.SchemaUpdate{
				Schema: `
					type Users {
						name: String
					}
				`,
			},
			testUtils.GenerateDocs{
			   AutoGenOptions{
			      NumberOfDocs: 10,
			   }
			},
			testUtils.Request{
			...
			},
```
At the point at which the test framework processes the GenerateDocs action it already knows which collections exist, and has access to their `[]client.Collection`.  There appears to be no reason to have new actions responsible for saving schema in the database in addition to the one we already have.

A setup like this would also automatically work with schema and collection patches.

You could then have an optional prop where consumers can optionally provide names or indexes of collections they wish to generate docs for (or just have a `gen.Option` for that).

Reply:
I like this idea very much. Wasn't sure myself if the current way if very optimal.
Will take care of it